Adyen/Webhooks/BalancePlatformWebhookHandler.cs

```diff
@@ -0,0 +1,187 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.Serialization;
+using Adyen.Model.ReportNotification;
+using Adyen.Model.ConfigurationNotification;
+using Adyen.Model.TransferNotification;
+using Adyen.Util;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Adyen.Webhooks
+{
+    public class BalancePlatformWebhookHandler
+    {
+        /// <summary>
+        /// Deserializes to a generic banking webhook from the <paramref name="jsonPayload"/>. Use this either to catch the
+        /// webhook type if unknown or if the explicit type is not required.
+        /// </summary>
+        /// <param name="jsonPayload">The json payload of the webhook.</param>
+        /// <returns>The parsed webhook packaged in a dynamic object.</returns>
+        /// <exception cref="JsonReaderException">Throws when json is invalid.</exception>
+        public dynamic GetGenericBalancePlatformWebhook(string jsonPayload)
```

Comment:
I've noticed that in the Java library it's called "BankingWebhook", one thing to take into consideration is naming-consistence.

I prefer the `BalancePlatformWebhook` as a consistent name across all libraries as it
However, if the other libraries already have "Banking" in their name, consider either changing it all to use "Banking"

I already see that documentation for example: https://docs.adyen.com/marketplaces-and-platforms/payment-event-webhooks are calling it "Transfer webhook" - I prefer a consistent naming approach here, definitely create a ticket for this :)